datumaro/components/annotation.py

```diff
@@ -178,40 +203,55 @@ def __eq__(self, other):
 
 @attrs(eq=False)
 class Mask(Annotation):
+    """
+    Represents a 2d single-instance binary segmentation mask.
+    """
+
+    BinaryMask = np.ndarray # 2d of type bool
```

Comment:
It's a bit unclear from the documentation, but I don't think type aliases are allowed at local scope. Instead, this declaration is treated as a variable of type `Type[np.ndarray]`. It's mentioned in [PEP 613](https://www.python.org/dev/peps/pep-0613/#id6), and in practice, mypy refuses to validate such usage. You can check it yourself by running mypy on this simplified example:

```python
import numpy as np

class C:
    x = np.ndarray

    def foo(self) -> x:
        ...
```

Reply:
Apparently, you're right about scoping. It's unfortunate, because I'm not sure I want such aliases in the global scope.